pixeltable/client.py

```diff
@@ -147,6 +150,174 @@ def create_table(
         _logger.info(f'Created table {path_str}')
         return tbl
 
+    def create_table_from_parquet(self, path_str: str, parquet_path: str,
+                                  explain_only : bool = False,
+                                  primary_key: Union[str, List[str]] = [],
+                                  num_retained_versions: int = 10) -> Optional[catalog.InsertableTable]:
+        """Create a new `InsertableTable` from a Parquet file. Requires pyarrow to be installed.
+
+        Args:
+            path_str: Path to the table.
+            parquet_path: Path to an individual Parquet file or directory of Parquet files.
+            explain_only : Print how the file will map to a table and return without creating the table.
+            primary_key: Primary key column(s).
+            num_retained_versions: Number of versions of the table to retain.
+
+        Returns:
+            The newly created table. The table will have loaded the data from the Parquet file(s)
+        """
+        import pyarrow as pa
+        from pixeltable.utils.arrow import get_pixeltable_schema, iter_tuples
+
+        input_path = Path(parquet_path).expanduser()
```

Comment:
The logic should definitely be in util/parquet.py

Reply:
ok. done.